threads/threads.yaml

```diff
@@ -0,0 +1,7 @@
+test:
+    tests:
+        - name: test_threads_rand
+          exec: test_thread_rand
+          type: unit
+          targets:
+              include: [armv7m7-imxrt106x]
```

Comment:
This test should be accessible on all hardware platforms. Marek mentioned this in his review.

Reply:
Previous Marek`s note was resolved with change from
```
targets:
    value: [armv7m7-imxrt106x]
```
to
```
targets:
    include: [armv7m7-imxrt106x]
```
as `value:` specifies the only architecture to run test on, and `include:` adds it to test targets.

Reply:
still not being run on ia32-generic